workflow/scripts/collect-fp-fn-benchmarks.py

```
@@ -0,0 +1,42 @@
+
+
+def load_data(path, callset):
+    d = pd.read_csv(path, sep="\t")
+    d.insert(0, "callset", callset)
+    return d
+
+
+results = pd.concat(
+    [
+        load_data(f, callset)
+        for f, callset in zip(snakemake.input.tables, snakemake.params.callsets)
```

Ruff (0.8.0)

Undefined name `snakemake`

(F821)